presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowSpecificationAABcp.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
```

Comment:
Also consider adding some tests to `AbstractTestQueries` class. This class is run as unit tests and it is much easier to debug. I would leave only one test as a product test and move rest of them to that class (this kind of approach is called a [TestPyramid](http://martinfowler.com/bliki/TestPyramid.html).